test/nctl/RPC.test.ts

```diff
@@ -86,6 +87,11 @@ describe('RPC', () => {
     }, 3 * 10000);
   });
 
+  xit('queryBalance', async () => {
```

Comment:
this tests is disabled, the whole situation is [different issue](https://github.com/casper-network/Governance/issues/361)